pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -589,7 +678,7 @@ private static void setupFileScanResultReceiver(Config cfg, String projectId) {
     private static void startNewSubscriberWithRecovery(ProjectSubscriptionName subscription,
                                                        Subscriber.Builder builder,
                                                        ExecutorProvider executorProvider,
-                                                       ExecutorService callbackExecutor,
+                                                       Executor callbackExecutor,
```

Comment:
A specific `ExecutorService` is not required here- an `Executor` instance suffices for the needs of this method.

Reply:
No longer applicable